src/utils/release-actions.ts

```diff
@@ -0,0 +1,39 @@
+import { ReleaseLabel } from '../consts/release';
+import { K8sQueryCreateResource } from '../k8s';
+import { ReleaseModel } from '../models';
+import { ReleaseKind } from '../types';
+import { generateNewReleaseName } from './release-utils';
+
+export const releaseRerun = (release: ReleaseKind, username: string) => {
+  const newName = generateNewReleaseName(release.metadata?.name ?? '');
+
+  const newRelease: ReleaseKind = {
+    ...release,
+    metadata: {
+      ...release.metadata,
+      creationTimestamp: undefined,
+      finalizers: undefined,
+      generateName: undefined,
```

Comment:
`metadata.generaeName` can be used for creating unique names with suffix. Not sure if there is some particular use case which we want to address